Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/AsyncPDFRenderer.mm

```diff
@@ -214,8 +216,12 @@
     return renderInfo.equivalentForPainting(currentRenderInfo);
 }
 
+#pragma mark TiledBackingClient
+
 void AsyncPDFRenderer::willRepaintTile(TiledBacking& tiledBacking, TileGridIdentifier gridIdentifier, TileIndex tileIndex, const FloatRect& tileRect, const FloatRect& tileDirtyRect)
 {
+    // FIXME [aprotyas]: Do we (or do we not) expect this to be called on the pending tile grid? Assert the behavior we want...
```

Comment:
I would say that the client should not get a `prepareContentForTile` and a `willRepaintTile` for initial tile creation. But if `prepareContentForTile` was issued and then state changed, a `willRepaintTile` could be issued.